### PR TITLE
Refactored powershell scripts

### PR DESCRIPTION
**Pull request by lokesh755**

### CODE REVIEW

File: images/win/Vs2017/scripts/Install-NodeLts.ps1 [hunk 1]
```diff
@@ -19,9 +19,6 @@ Add-MachinePathItem $PrefixPath
 Add-MachinePathItem $CachePath
```
Comment (chrisrpatterson) [inline images/win/Vs2017/scripts/Install-NodeLts.ps1]: I don't think this needs to be in the path only the prefix does.  We should also set NPM_CONFIG_CACHE and NPM_CONFIG_PREFIX as system level environment variables with the appropriate values.

File: images/win/Vs2017/scripts/Validate-ServiceFabricSDK.ps1 [hunk 2]
```diff
@@ -0,0 +1,18 @@
+################################################################################
```
Comment (chrisrpatterson) [inline images/win/Vs2017/scripts/Validate-ServiceFabricSDK.ps1]: @NCarlsonMSFT can you review this.

File: images/win/Vs2017/scripts/Install-ServiceFabricSDK.ps1 [hunk 3]
```diff
@@ -12,5 +12,3 @@ New-Item -Path $temp_install_dir -ItemType Directory -Force
 Set-ExecutionPolicy RemoteSigned -Scope CurrentUser -Force
 
 WebpiCmd.exe /Install /Products:MicrosoftAzure-ServiceFabric-CoreSDK /AcceptEula
```
Comment (chrisrpatterson) [inline images/win/Vs2017/scripts/Install-ServiceFabricSDK.ps1]: @NCarlsonMSFT  can you review this

Comment (NCarlsonMSFT) [reply images/win/Vs2017/scripts/Install-ServiceFabricSDK.ps1]: Double checked that is the right product ID

**Pull request merged**